### ISSUE TITLE
Release 0.15 and bower

### ISSUE THREAD

**Issue by b-durand** [powmedia/backbone-forms#399]
Hello, can you tag the version 0.15.0 for bower? The tag 0.14.0 is released on 28 november 2013! Some features are missing like the submit event.

Thanks.

**Comment by htrex** [comment]
you can always checkout the master branch with:
```
bower install https://github.com/powmedia/backbone-forms.git#master
```
or target a specific commit release with something like:
```
bower install https://github.com/powmedia/backbone-forms.git#22c22e09f0
```

**Comment by rjsteinert** [comment]
The lack of releases does make this project feel a bit abandoned.

**Comment by glenpike** [comment]
Same as #418 - fixed by #487
